prism-languages.php

```diff
@@ -5,7 +5,7 @@
  * @return array Hash of lang => language
  */
 function mkaz_code_syntax_block_get_supported_languages() {
-	
+
 	$languages = array(
 		"apacheconf" => "Apache Config",
 		"bash" => "Bash/Shell",
```

Comment:
:warning: **Warning**: Array double arrow not aligned correctly; expected 7 space(s) between &quot;&quot;bash&quot;&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).